modules/exploits/linux/snmp/awind_snmp_exec.rb

```diff
@@ -0,0 +1,168 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'openssl'
+require 'base64'
```

Comment:
Have you tried running this without the `require` statements? I'm 95% sure these would have been included previously.